keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ApplicationLoadBalancerModel.kt

```diff
@@ -34,7 +34,7 @@ data class ApplicationLoadBalancerModel(
   data class Action(
     val type: String,
     val order: Int,
-    val targetGroupName: String,
+    val targetGroupName: String?,
```

Comment:
Is it possible to default it to something? or it is completely meanless?

Reply:
My instinct is to have these models be as faithful as possible to the underlying JSON representation, because it makes it easier to figure out how the data was populated if something goes wrong.

Reply:
makes sense.